### DISCUSSION TITLE
Why would web3modal successfully use a mobile link to connect to a wallet, but sign client does not?

### DISCUSSION THREAD

**Discussion by The18thWarrior** [WalletConnect/walletconnect-monorepo#4148] [Developer Support]
I am working on a react-native mobile wallet. I have had WC setup for awhile, but recently I am unable to connect to the wallet on a device it's installed on. https://web3modal.com works as expected, but for some reason https://react-app.walletconnect.com/ does not. So apps like Uniswap aren't seeing my wallet. Any ideas on how this would happen?

**Comment by The18thWarrior** [comment]
Upon further investigation, it looks like WalletConnectModal is what isn't working now. Is this expected?

**Comment by The18thWarrior** [comment]
For those that want to know, it was a combination of adding the wc scheme to my AndroidManifest and a change in the format of the invoked urls in when triggering linking into the app.

**Comment by Avkubs** [comment]
This is interesting

**Comment by Dogi340** [comment]
good wertt
